.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+# Type of change
```

Comment:
Right! I didnt notice a CHANGELOGS.md which led me to believe semantic commits were for primarily for aesthetic purposes
But yea, it sounds better if we skip this one or better, change this to "Please include a brief summary of the change." since the PR title might not be enough sometimes